src/fah/client/win/WinOSImpl.cpp

```diff
@@ -203,7 +203,13 @@ void WinOSImpl::init() {
 }
 
 
-const char *WinOSImpl::getName() const {return "win32";}
+const char *WinOSImpl::getName() const {
+#ifdef _WIN64
+  return "win64";
+#else
```

Comment:
`win32` isn't the only other option; this ignores 16 and ARM. It'd probably be simpler to just return `win64` instead, since it's only compiled for 64bit computers.